core/src/main/java/org/apache/accumulo/core/client/Connector.java

```diff
@@ -313,6 +313,23 @@ public abstract ConditionalWriter createConditionalWriter(String tableName,
    */
   public abstract ReplicationOperations replicationOperations();
 
+  /**
+   * @return {@link ConnectionInfo} which contains information about Connection to Accumulo
+   * @since 2.0.0
+   */
+  public abstract ConnectionInfo info();
+
+  /**
+   * Creates new Connector for new user with principal and token
```

Comment:
Could also replace `new user` with `another user`

Reply:
Removed this method 368c222c543eba07